TLS_E2E_Automation_Framework/cypress/e2e/myAccount.spec.js

```diff
@@ -0,0 +1,40 @@
+import * as loginLogout from "../e2e/tls_page_elements/loginLogout.js";
+import * as myAccount from "../e2e/tls_page_elements/myAccount.js";
+
+const environment = Cypress.env("ENV") || "prod";
+const url = Cypress.env(`${environment}_url`);
+const myAccountLink =
+  "//*[@id='tls-home-page-root']/div/div/nav/div/div[2]/span[5]/a";
```

Comment:
@kavin1711 instead of using xpaths everywhere we can make best practice to use where ever possible like here,
.tls-header-navigation__right-controls .tls-header-navigation__my-account  or .tls-header-navigation__right-controls > span

Reply:
i tried this i didnt work, then opted for xpath